src/components/SectionStep/styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from "@emotion/styled";
+
+const style = (Component) => styled(Component)`
+
+`;
+
+export default style;
```

Comment:
If there are no styles, we don't need a styles file.

Reply:
Fixed in 2e1b15a